src/stylis-rtl.test.js

```diff
@@ -3,7 +3,7 @@ import { compile, middleware, prefixer, serialize, stringify } from 'stylis';
 import stylisRtlPlugin from './stylis-rtl';
 
 const stylis = (css, extraPlugins = []) =>
-  serialize(compile(css), middleware([stylisRtlPlugin, ...extraPlugins, stringify]));
+  serialize(compile(css), middleware([...extraPlugins, stylisRtlPlugin, stringify]));
```

Comment:
requiring this order for the prefixer plugin is unfortunate - but this order makes IMHO more sense (for the prefixer within `extraPlugins`), let's just do prefixing first which can insert new nodes (sort of) and then do the flipping

You could try to handle this for the other order as well, but I personally have no energy for it so this is what I'm proposing. It can be seen as a breaking change though - so u need to decide how to best handle this.